tools/install.sh

```diff
@@ -267,27 +267,27 @@ elif [ "$LINUX_VERSION" == "BackBox" ] || \
 	 [ "$LINUX_VERSION" == "Debian"  ] || \
 	 [ "$LINUX_VERSION" == "Kali"    ] || \
 	 [ "$LINUX_VERSION" == "Ubuntu"  ]; then
-	apt-get install -y libfreetype6-dev python-dev python-pip
+	apt-get install -y libfreetype6-dev python3-dev python3-pip
 	if [ -z "$KING_PHISHER_SKIP_CLIENT" ]; then
 		apt-get install -y gir1.2-gtk-3.0 gir1.2-gtksource-3.0 \
-			gir1.2-webkit-3.0 python-cairo libgeos++-dev \
-			libgtk-3-dev libpq-dev python-gi python-gi-cairo libpq-dev \
-			python-gobject python-gobject-dev python-paramiko pkg-config
+			gir1.2-webkit-3.0 python3-cairo libgeos++-dev \
+			libgtk-3-dev libpq-dev python3-gi python3-gi-cairo libpq-dev \
+			python-gobject python-gobject-dev pkg-config
```

Comment:
Should this be `python3-gobject python3-gobject-dev pkg-config`? Otherwise are those packages for both 2.x and 3.x or are they unnecessary?